content/docs/pulumi-cloud/admin/self-hosted/airgapped.md

```diff
@@ -0,0 +1,131 @@
+---
+title_tag: Deploying into Air-Gapped Environments | Self-Hosting Pulumi
+meta_desc: Learn how to deploy Pulumi self-hosted into air-gapped environments.
+title: Air-Gapped
+h1: Deploying Pulumi Self-Hosted into Air-Gapped Environments
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    cloud:
+        name: Air-Gapped
+        parent: pulumi-cloud-admin-self-hosted
+        weight: 5
+        identifier: pulumi-cloud-admin-self-hosted-airgapped
+---
+
+{{% notes type="info" %}}
+Self-hosting is only available with **Pulumi Business Critical**. If you would like to evaluate the self-hosted Pulumi Cloud, sign up for the [30-day trial](/product/self-hosted#self-hosted-trial) or [contact us](/contact/).
+{{% /notes %}}
+
+For organizations operating in highly regulated industries or environments with strict security requirements, deploying cloud infrastructure in an air-gapped environment is often a requirement. Such environments do not have network connectivity with the outside world, which many of Pulumi's default workflows assume.
+
+Pulumi can be configured to run in air-gapped environments through [self-hosting](/docs/pulumi-cloud/admin/self-hosted), enabling enterprises to manage infrastructure as code securely within their private networks, remaining compliant while gaining the benefits of modern infrastructure automation.
+
+In this guide, we will explore how to deploy Pulumi Self-Hosted in an air-gapped environment, covering the key requirements, setup process, and best practices.
+
+## Why Deploy Pulumi in an Air-Gapped Environment?
+
+Air-gapped environments are completely isolated from the public internet, for scenarios where data security and compliance are paramount. Common use cases include:
+
+* **Financial Services**: Ensuring sensitive transaction data remains within internal networks.
+* **Healthcare**: Complying with HIPAA and other patient data protection regulations.
```

Comment:
Maybe I'm being paranoid so ignore if so.
But I fear that someone reading this item (or the list in general) will think they have to go with airgapped Pulumi if they are in the Healthcare industry (or other industries in the list).
So I wonder if some text should be added to lighten that implication.
Or just ignore this comment if you think I'm worrying too much.

Reply:
Agree with you. I will wordsmith this.